src/cdk/drag-drop/drag.spec.ts

```diff
@@ -2229,6 +2239,24 @@ class StandaloneDraggable {
   endedSpy = jasmine.createSpy('ended spy');
 }
 
+@Component({
+  template: `
+    <svg><g
+      cdkDrag
+      (cdkDragStarted)="startedSpy($event)"
+      (cdkDragEnded)="endedSpy($event)"
```

Comment:
The start and end events can be removed since they aren't being used in any of the tests.

Reply:
Done